content/SCALE/SCALECLIReference/Account/CLIgroup.md

```diff
@@ -10,8 +10,221 @@ tags:
+{{< /hint >}}
+
+A space is required between the provided argument and entered data, for example `name: test_group`.
+
+Press <kbd>F2</kbd> or click **Save** to save the modified file.
+
+Press <kbd>F10</kbd>, <kbd>Esc</kbd>, or click **Quit** to exit the TUI.
+The `create` command automatically executes upon exit.
+{{< /expand >}}
+
+### Create Configuration Arguments
+
+`create` has 7 available arguments for group configuration. They are:
+
+* gid
+* name
+* smb
+* sudo_commands
+* sudo_commands_nopasswd
+* allow_duplicate_gid
+* users
+
+{{< expand "Argument Functions and Examples" "v" >}}
+{{< truetable >}}
+| Argument | Type | Function | Required? |
+|-----------|-------------|-------------|-------------|
+| `gid=` | Integer | Assigns the group identification number. If `gid` is not provided it is automatically filled with the next one available. <br> Ex. `gid=3005` <br> &emsp; Where `3005` is an available GID number. | No |
+| `name=` | String | Sets the group name. <br> Ex. `name=test_group` <br> &emsp; Where `test_group` is the desired group name. | Yes |
+| `smb=` | Boolean | Sets whether the group should be mapped into an NT group for Windows SMB sharing. Defaults to `true`. <br> Ex. `smb=false` <br> &emsp; Where `false` is a boolean value. | No |
+| `sudo_commands=` | Array | Sets any sudo commands group members are allowed to use. Security best practice is to limit sudo permissions to administrative users. <br> Ex. `sudo_commands="</path1/>,</path2/>"` <br> &emsp; Where `</path1/>` and `</path2/>` are absolute paths to the location of executable scripts or packages. <br> &emsp; You can also use `sudo_commands="ALL"` | No |
+| `sudo_commands_nopasswd=` | Array | Sets any sudo commands group members are allowed to use without entering a password. Exercise caution when allowing sudo commands without password prompts. It is recommended to limit this privilege to trusted users and specific commands to minimize security risks. <br> Ex. `sudo_commands_nopasswd="</path1/>,</path2/>"` <br> &emsp; Where `</path1/>` and `</path2/>` are absolute paths to the location of executable scripts or packages. <br> &emsp; You can also use `sudo_commands_nopasswd="ALL"`, but this is not recommended. | No |
+| `allow_duplicate_gid=` | Boolean | If set to true, allows distinct group names to share the same group identification number. Defaults to false. <br> Important: Use only if absolutely necessary. Duplicate GIDs can lead to unexpected behavior. <br> Ex. `allow_duplicate_gid=true` <br> &emsp; Where `true` is a boolean value. | No |
+| `users=` | Array | Assigns users to the group with a list of user identification numbers (UIDs). <br> Ex. `users=[3001,3002]` <br> &emsp; Where `3001` and `3002` are UID numbers to assign to the group. | No |
+
+{{< /truetable >}}
+{{< /expand >}}
+
+## Delete Command
+The `delete` command deletes an existing group.
+
+`delete` has one command option.
```

Comment:
What is the one option this command has? Include it here, and also in the command examples provided next. If it is required, then include it in the examples.

Reply:
It's in the expand snippet for now (I think it's to delete group members at the same time?) but this might get reworked when I get to that one.